src/content/energy_consumption/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Ethereum Energy Consumption
+description: The basic information you need to understand Ethereum's energy consumption.
+lang: en
+sidebar: true
+---
+
+# Introduction to Ethereum's Energy Consumption {#introduction-to-ethereum-energy-consumption}
+
+Ethereum is now on the cusp of becoming a truly low-carbon blockchain. Achieving this without sacrificing decentralization has been a major technical challenge for Ethereum developers. This page will explain why building out Ethereum to its current state has required a significant upfront carbon cost and how this will be remediated in future ugrades.
```

Comment:
As we agreed in DMs - transparency and showing awareness to the problem early is good

```suggestion
Ethereum's current energy expenditure is too high and unsustainable. Resolving energy expenditure concerns without sacrificing security and decentralization has been a significant technical challenge for Ethereum's development. Let's explore why building Ethereum has had a high environmental impact and how upcoming network upgrades will dramatically change this.
```